packages/esm-service-queues-app/src/patient-queue-metrics/metrics-card.component.tsx

```diff
@@ -45,7 +45,7 @@ const MetricsCard: React.FC<MetricsCardProps> = ({
             <div className={styles.link}>
               <ConfigurableLink
                 className={styles.link}
-                to={`\${openmrsSpaBase}/queue-list/${service}/${serviceUuid}/${locationUuid}/`}>
+                to={`\${openmrsSpaBase}/home/service-queues/queue-list/${service}/${serviceUuid}/${locationUuid}/`}>
```

Comment:
It's preferable to move this into a separate variable. Also, prefer using `window.getOpenmrsSpaBase()` like so:

```tsx
const queueListPath = window.getOpenmrsSpaBase() + `home/service-queues/queue-list/${service}/${serviceUuid}/${locationUuid}`
```

Note that `window.getOpenmrsSpaBase()` appends a `/` at the end.